crates/sui-rpc-benchmark/src/bin/pull_grafana_logs.rs

```diff
@@ -90,6 +90,58 @@ fn extract_from_message(message: &str) -> Option<LogEntry> {
     None
 }
 
+/// Example log format:
+/// 2025-08-10T00:05:45.423808Z  INFO sui_indexer_alt_jsonrpc_proxy::handlers: Request: Method: suix_getAllBalances, Params: ["0x10e8a57972082d89f8e2a31589a96da4a0ade2ac003e4f41a0f7b77dbfd752ba"] | Response body (UTF-8): {"jsonrpc":"2.0","id":47539,"result":[...]}
+fn extract_from_proxy_message(message: &str) -> Option<LogEntry> {
```

Comment:
Since we are parsing structed logs are string operations favored over regex here because of performance reasons?

Reply:
Not really actually. I mainly just followed the existing logic in `extract_from_message`. Using regex here would be cleaner indeed. I'll explore it in a followup - thanks for the suggestion!